core/pom.xml

```diff
@@ -153,6 +153,12 @@
       <artifactId>jzlib</artifactId>
       <version>1.1.3</version>
     </dependency>
+    <dependency>
+      <groupId>org.hibernate</groupId>
+      <artifactId>hibernate-validator</artifactId>
+      <version>6.0.10.Final</version>
+    </dependency>
+
```

Comment:
to avoid them the temptation to rely on hibernate-validator internals this would indeed work, but won't prevent classpath conflict if they decide to have another version of hibernate-validator as a dependency

that being said, hibernate-validator internal classes are hardly usable out of validation-api context.

Reply:
we actually rely on ParameterMessageInterpolator. Could probably be re-implemented